Assets/HoloToolkit/Input/Scripts/Utilities/AttachToController.cs

```diff
@@ -13,27 +13,19 @@ namespace HoloToolkit.Unity.InputModule
     /// </summary>
     public class AttachToController : ControllerFinder
     {
-#if UNITY_WSA && UNITY_2017_2_OR_NEWER
-        [Header("AttachToController Elements")]
-        [SerializeField]
-        protected new InteractionSourceHandedness handedness = InteractionSourceHandedness.Left;
-#endif
-        [SerializeField]
-        protected new MotionControllerInfo.ControllerElementEnum element = MotionControllerInfo.ControllerElementEnum.PointingPose;
-
         public bool SetChildrenInactiveWhenDetached = true;
 
         [SerializeField]
-        protected Vector3 positionOffset = Vector3.zero;
+        protected Vector3 PositionOffset = Vector3.zero;
```

Comment:
From what I can tell, we've mostly been using lower camel case for protected names in the Toolkit.

Reply:
Just following [MSDN naming conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-fields)

Reply:
Consistency seems more important, but we should change them all if we want to follow those naming conventions.

Reply:
Well as far as I can tell, I'm the single biggest contributor and I follow the MSDN guidelines. 
So, it might be better to follow industry standard and fix this in a separate issue.

Reply:
From a *very* quick, non-scientific search, I found 17 files that use PascalCase, 17 files that use camelCase (not counting the SWIG-generated files, which all use camelCase), and 2 files that use underscores.